app/src/test/java/com/example/tanks/engine/GameThreadTest.java

```diff
@@ -0,0 +1,91 @@
+package com.example.tanks.engine;
+
+import android.graphics.Canvas;
+import android.support.annotation.NonNull;
+import android.view.SurfaceHolder;
+import android.view.SurfaceView;
+
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.mockito.Mockito.atLeastOnce;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by Marcin on 23.10.2017.
+ */
+public class GameThreadTest {
+
+    @Mock
+    SurfaceView surfaceView;
+
+    @Mock
+    SurfaceHolder surfaceHolder;
+
+    @Mock
+    Canvas canvas;
+
+    @InjectMocks
```

Comment:
to jest chyba tu nie potrzebne jak robie inicjalizacje w metodzie `givenStartedGameThread()`